docs/get-started/claims.md

```diff
@@ -1,15 +1,12 @@
 # <b>CLAIMS</b>
 ---
 
-The Kusari network is SwapDex's experimental, community-focused research & development network. If you held the SDX ERC-20 token or SDX coins until **Friday, December 3rd 2021**, you are entitled to claim KSI on the Kusari network.
+If you held the SDX ERC-20 token or SDX coins until **Friday, February 4th 2022**, you are entitled to claim SDX on the SwapDex network.
 
 !!! hint
     Watch our <a href="https://youtu.be/lPRF43-9IGU" target="_blank"> YouTube Walkthrough </a>
 
-You can claim KSI by visiting our <a href="https://app.kusari.network/" target="_blank"> Dashboard Claim Function </a>.
-
-!!! hint
```

Comment:
you can leave the hint in and change the URL to https://sdxsnapshot.herokuapp.com/